src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix3x2.cs

```diff
@@ -799,9 +799,10 @@ public static Matrix3x2 Multiply(Matrix3x2 value1, float value2)
         /// <returns>The hash code.</returns>
         public override readonly int GetHashCode()
         {
-            return unchecked(M11.GetHashCode() + M12.GetHashCode() +
-                             M21.GetHashCode() + M22.GetHashCode() +
-                             M31.GetHashCode() + M32.GetHashCode());
+            return HashCode.Combine(
```

Comment:
The `GetHashCode` can be omitted, as `HashCode.Combine` itself takes the hashcode. So it's done twice.

Reply:
Right, forgot about that, thanks for calling this out!

Reply:
agree, but there are the same in Vectors. Can I change it as well in this PR?

Reply:
I'm fine with `Vector2/3/4` also being updated as part of this PR.

@stephentoub, @terrajobst Do we want to file an issue for an analyzer for this scenario?

Reply:
can't use `var hash = new HashCode();` because `Do not use default value type constructor`

Reply:
You can use `HashCode hash = default`.

Reply:
> Do we want to file an issue for an analyzer for this scenario?

There are trade-offs, and I'm not sure which is better.  By using GetHashCode at the call site, we'll likely reduce the number of generic instantiations of HashCode.Combine used, since everything will be int-based.